base/flux/deployment.yaml

```diff
@@ -10,7 +10,7 @@ spec:
   template:
     metadata:
       annotations:
-        prometheus.io.scrape: "true"
+        prometheus.io/port: "3031"
```

Comment:
You need to set this port in Flux args with `--listen-metrics=:3031` and please enable scraping.

Reply:
Added the `--listen-flag` and renabled `prometheus.io.scrape` . Out of curiosity is there a reason why scrape is not enabled in the main [Flux deployment] (https://github.com/fluxcd/flux/blob/master/deploy/flux-deployment.yaml)?

Reply:
For consistency please rename it to `prometheus.io/scrape`. Thanks

Reply:
Just pushed the fixed annotation :)